api/UVIndex.js

```diff
@@ -14,6 +14,24 @@ router.post("/",function(req,res){
 	};
 	if(city && state){
 	  controllerObj.getUVIndexes(location,retval,function(val){
+		// group the UV indexes into categories
+		val.map(function(result){
+			if(result){
+				var UV_VALUE = result.UV_VALUE;
+				if(UV_VALUE < 3)
+					result['EXPOSURE'] = "Low";
+				if(UV_VALUE >= 3 && UV_VALUE <= 5)
+					result['EXPOSURE'] = "Moderate";
+				if(UV_VALUE >= 6 && UV_VALUE <=7)
+					result['EXPOSURE'] = "High";
+				if(UV_VALUE >=8 && UV_VALUE <=10)
+					result['EXPOSURE'] = "VeryHigh";
+				if(UV_VALUE >= 11)
+					result['EXPOSURE'] = "Extreme";
```

Comment:
Expected '{' and instead saw 'result'.<br>Mixed double and single quotes.<br>['EXPOSURE'] is better written in dot notation.